modules/core/TextureRings.cpp

```diff
@@ -1,26 +1,42 @@
-//
-//  Rings.cpp
-//  Demo_AreaLight
-//
 //  Created by Mahmoud El Bergui on 18.04.22.
 //
 
 #include "TextureRings.h"
+#include"PerlinNoise.h"
```

Comment:
Do you use Perlin Noise in this texture?

Reply:
No I don't I was just doing some tests and forgot to put take it off